index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Product Landing Page</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      Explore
+      Start a project
+      <img src="logo.png">
+      <input type="text" placeholder="Search">
+      Log in
+    </nav>
+    <h1>The Minimalist’s Wall Calendar | Visualize Your Year</h1>
+    <p>Minimalist, forgiving, and beautiful. A wall calendar designed to make 2021 feel like a breath of fresh air.</p>
+  </header>
+  <main>
+    <figure>
+      <img src="hero.png">
+      <figcaption>Project we Love | Product Design | Cambridge, MA</figcaption>
+    </figure>
+    <aside>
+      <hr>
+      <p>
+        $13,757<br>
+        pledged of $1,200 goal
+      </p>
+      <p>
+        215 <br>
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <input type="button" value="Back this Project"><br>
+      <input type="button" value="Remind me">
+      <p>
+        <sub>All or nothing. This project will only be funded if it reaches its goal by Thu, November 26 2020 1:35 PM
+          CST.</sub>
+      </p>
+    </aside>
+    <nav>
+      Campaign | FAQ | Updates<sup>2</sup> | Comments<sup>19</sup> | Community
+    </nav>
+    <aside>
+      <nav>
+        <p>STORY</p>
+        <hr>
+        <p>RISKS</p>
+        <hr>
+        <p>ENVIRONMENTAL<br>
+          COMMITMENTS</p>
+      </nav>
```

Comment:
Wrapping the individual elements in `p` tags is slightly better, however `p` tags are meant to represent a paragraph of text, and not items that indicate the structure of the page or navigate you somewhere.

I would read over the comments left on the other `nav` tags above and consider what a better alternative tag here would be.

Additionally, an `hr` tag represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section. In this case, there is no thematic difference between each navigation item, they're all navigating to a different part of the page, so I believe it's probably inappropriate in this context.

If what you're shooting for _aesthetically_ is a line between the elements, you can actually do that with CSS.

Repeat this to yourself: HTML has nothing to do with how things look, and only what things mean.

If you ever find yourself adding an HTML element just for the look of the element....don't.